docs/source/package_orderers.rst

```diff
@@ -0,0 +1,86 @@
+================
+Package Orderers
+================
+
+Rez's default version resolution algorithm will always sort by the latest alphanumeric
+version. However, package orderers allow you to customize this functionality globally,
+or at a per package level.
+
+Configuring package orderers
+============================
```

Comment:
```suggestion
```
I don't think we need a section for configuring.

Reply:
@JeanChristopheMorinPerso Do you mean we don't need this section because it already exists on the `Configuring Rez` page? I'm not really adding new content here, just cross referencing to that page.